src/components/Reown/modals/SendTransactionModal.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect } from 'react';
+import { t } from 'ttag';
+import { useSelector, useDispatch } from 'react-redux';
+import { constants, numberUtils } from '@hathor/wallet-lib';
+import { unregisteredTokensDownloadRequested } from '../../../actions';
+import { CopyButton } from '../../CopyButton';
+
+export function SendTransactionModal({ data, firstAddress, onAccept, onReject }) {
```

Comment:
Why do you need `firstAddress` in this component?

Reply:
It was unused, removed it, thanks